subprojects/docs/src/samples/native-binaries/cpp-exe/build.gradle

```diff
@@ -20,7 +20,7 @@ model {
                 dependsOn binary.tasks.link
                 doFirst {
                     if (binary.toolChain in Gcc) {
-                        ["strip", binary.tasks.link.outputFile].execute()
+                        ["strip", binary.tasks.link.outputFile].execute().waitFor()
```

Comment:
Let's set a timeout so we don't end up with a hung test case potentially:

http://docs.oracle.com/javase/8/docs/api/java/lang/Process.html#waitFor-long-java.util.concurrent.TimeUnit-